Toplean_Alexandru/week-02/Domain/Domain/GetOrdersOp/GetOrdersCmd.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Domain.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Domain.Domain.GetOrdersOp
+{
+    public struct GetOrdersCmd
+    {
+        public List<Order> Orders { get; }
+        public Restaurant Restaurant { get; }
+
+        public GetOrdersCmd(Restaurant restaurant)
+        {
+            Restaurant = restaurant;
+            Orders = restaurant.Orders;
```

Comment:
Probably there's a bit of confusion here, the command itself should not contain the actual result of the operation. Yes, it takes a restaurant as an input, but it's the Ops responsibility to resolve the orders and return them (not command's responsibility).